pkg/venafi/tpp/connector_test.go

```diff
@@ -1981,7 +1981,7 @@ func TestReadPolicyConfiguration(t *testing.T) {
 				[]string{".*"},
 				[]string{".*"},
 				[]endpoint.AllowedKeyConfiguration{
-					{certificate.KeyTypeRSA, certificate.AllSupportedKeySizes(), nil},
+					{certificate.KeyTypeRSA, []int{1024, 2048, 3072, 4096, 8192}, nil},
```

Comment:
Could you revert this, as it was better set as it was before.  Also I have question. Is this size allowed in TPP 22.3? Iif not, we will need to upgrade, since running test got the following:
```
        get:    {SubjectCNRegexes:[^([\p{L}\p{N}-*]+\.)*vfidev\.com$ ^([\p{L}\p{N}-*]+\.)*vfidev\.net$ ^([\p{L}\p{N}-*]+\.)*vfide\.org$] SubjectORegexes:[^Venafi Inc\.$] SubjectOURegexes:[^Integration$] SubjectSTRegexes:[^Utah$] SubjectLRegexes:[^Salt Lake$] SubjectCRegexes:[^US$] AllowedKeyConfigurations:[{KeyType:0 KeySizes:[2048 3072 4096 8192] KeyCurves:[]}] DnsSanRegExs:[^([\p{L}\p{N}-*]+\.)*vfidev\.com$ ^([\p{L}\p{N}-*]+\.)*vfidev\.net$ ^([\p{L}\p{N}-*]+\.)*vfide\.org$] IpSanRegExs:[.*] EmailSanRegExs:[.*] UriSanRegExs:[.*] UpnSanRegExs:[.*] AllowWildcards:true AllowKeyReuse:true} 
        expect: {SubjectCNRegexes:[^([\p{L}\p{N}-*]+\.)*vfidev\.com$ ^([\p{L}\p{N}-*]+\.)*vfidev\.net$ ^([\p{L}\p{N}-*]+\.)*vfide\.org$] SubjectORegexes:[^Venafi Inc\.$] SubjectOURegexes:[^Integration$] SubjectSTRegexes:[^Utah$] SubjectLRegexes:[^Salt Lake$] SubjectCRegexes:[^US$] AllowedKeyConfigurations:[{KeyType:0 KeySizes:[2048 4096 8192] KeyCurves:[]}] DnsSanRegExs:[^([\p{L}\p{N}-*]+\.)*vfidev\.com$ ^([\p{L}\p{N}-*]+\.)*vfidev\.net$ ^([\p{L}\p{N}-*]+\.)*vfide\.org$] IpSanRegExs:[.*] EmailSanRegExs:[.*] UriSanRegExs:[.*] UpnSanRegExs:[.*] AllowWildcards:true AllowKeyReuse:true}
```

Reply:
Looks like this keysize was already supported in 2021: https://github.com/Venafi/vcert/pull/164/files#diff-f30f61745dc1d11a1a2455b36477d3840e36d4b66e3f6c32854db74c9efd51faR13

Reply:
I reverted the change and fixed the failing test.